plugins/modules/win_partition.py

```diff
@@ -91,9 +97,9 @@
     partition_size: 5 GiB
     disk_number: 1
 
-- name: Resize previously created partition to it's maximum size and change it's drive letter to E
+- name: Resize previously created partition to it's maximum size and change it's mountpoint to E:\SQL_Log
   community.windows.win_partition:
-    drive_letter: E
+    drive_letter: E:\SQL_Log
```

Comment:
since this is now a mountpoint, should `drive_letter` be renamed? or a new `mountpoint` variable added?

Reply:
I think replacing the `drive_letter` variable with `mountpoint` is the better choice since drive letters are actually mountpoints as well.

Reply:
Should I upload a modified version based on this?

Reply:
Removing the `driver_letter` seems to be a breaking change, so at least, the module major version needs to be bumped if `driver_letter` is removed. Maybe supporting the two is more appropriate. And maybe there's a way to automatically alias `drive_letter` to `mountpoint`?

Please note that I'm not a maintainer on this repository. I just started using this today :-)

Reply:
Then the maintainers will decide I guess. Frankly, I don't really see the point of adding an alias that logically does the exact same thing, but I see your point that with this feature added the `drive_letter` variable name is under-representing the option it is used for. At the other hand a replace could be easily done with a ctrl+h and it wouldn't need any additional change in the code or in the documentation.

Reply:
We shouldn't make a breaking change but what we can do is change the option to `mountpoint` to better reflect what it is used for but add an alias for `drive_letter` so that existing playbooks will continue to work. The alias is key to ensuring that people using this module don't need to deal with a breaking change, especially one where there's no in between where both options work.